ckanext/datavic_iar_theme/templates/package/snippets/resource_form.html

```diff
@@ -1,36 +1,111 @@
-{% ckan_extends %}
```

Comment:
Copied the whole resource_form.html from the core in order to place required_message in the top of the form but outside the existing fields and action blocks.